apps/condo/domains/subscription/constants/index.js

```diff
@@ -1,11 +1,57 @@
+const { values, without } = require('lodash')
 const SUBSCRIPTION_TRIAL_PERIOD_DAYS = 15
 
 const SUBSCRIPTION_TYPE = {
     DEFAULT: 'default',
     SBBOL: 'sbbol',
 }
 
+/**
+ * Reduced set of statuses from a set of statuses in external system, that contains much more of them.
+ * Based on this status a system will filter payment request for subsequent fetching of statuses from remote system.
+ */
+const SUBSCRIPTION_PAYMENT_STATUS = {
```

Comment:
@SavelevMatthew look at this too

Reply:
It is deviated from set of statuses in architectural presentation.
I think, this set is describes what happens with payment in a more detailed way.
Real example of payment statuses from bank is attached in Jira for this task.

Reply:
It's differs from Acquiring MultiPayment ones, since it's only uses first 4 out of 6 listed. For cancelled you can use deletedAt, but for stopped idk